packit/config/aliases.py

```diff
@@ -16,10 +16,10 @@
 ALIASES: Dict[str, List[str]] = {
     "fedora-all": ["fedora-35", "fedora-36", "fedora-rawhide"],
```

Comment:
When F36 was branched, you added it here too. For the sake of consistency, could you add F37?